test/parallel/test-tls-ticket.js

```diff
@@ -27,14 +30,25 @@ function createServer() {
   }, function(c) {
     serverLog.push(id);
     c.end();
+
+    counter++;
+
+    // Rotate ticket keys
+    if (counter === 1) {
+      previousKey = server.getTicketKeys();
+      server.setTicketKeys(crypto.randomBytes(48));
+    } else if (counter === 2) {
+      server.setTicketKeys(previousKey);
+    }
```

Comment:
Is `counter` ever expected to be `> 2`? If not, can we add an `UNREACHABLE()` throw in and else?